frontend/src/lib/components/apps/components/inputs/AppMultiSelectV2.svelte

```diff
@@ -30,49 +32,59 @@
 
 	const { app, worldStore, selectedComponent, componentControl } =
 		getContext<AppViewerContext>('AppViewerContext')
-	let items: string[]
 
 	const resolvedConfig = initConfig(
-		components['multiselectcomponent'].initialData.configuration,
+		components['multiselectcomponentv2'].initialData.configuration,
 		configuration
 	)
+	const outputs = initOutput($worldStore, id, { result: [] as any[] })
```

Comment:
result is still string[]

Reply:
Allowing value to be any[] looks really cool to me, do we need results to be string[] still?

Reply:
d04620e86e416b13885828c87dc9f4b8311f5b48